api/api_types.go

```diff
@@ -32,6 +33,17 @@ type ElectionSummary struct {
 	Results      [][]*types.BigInt `json:"result,omitempty"`
 }
 
+type ElectionResults struct {
+	// ABIEncoded is the abi encoded election results
+	ABIEncoded string `json:"abiEncoded"`
+
+	CensusRoot types.HexBytes `json:"censusRoot"`
+	// SourceContractAddress is the address of the smart contract containing the census
+	OrganizationID        types.HexBytes `json:"organizationId"`
+	Results               [][]*big.Int   `json:"results"`
+	SourceContractAddress types.HexBytes `json:"sourceContractAddress"`
```

Comment:
this field should have the JSON decorator `omitempty` since it won't be always available.